### ISSUE TITLE
challenge_reward_error: can't move tokens to blobber: uint64 overflows int64

### ISSUE THREAD

**Issue by boddumanohar** [0chain/0chain#1763]
we are seeing a errors of this kind in smart contract transactions of type `challenge_response` to check the details, one can log into events_db and run this sql query

```
select transaction_output from transactions where transaction_data like '%' || 'challenge_response' || '%' and status <> 1;
```

within the transactions table, we can also look into `transaction_data` and other fields to know more details about the transactions

**Comment by boddumanohar** [comment]
@peterlimg  it looks like there are still still overflow errors in other storage sc transactions.

**Comment by peterlimg** [comment]
@boddumanohar I'll have a check

**Comment by peterlimg** [comment]
closed via #2145
